src/backend/main/go.backends/store/cassandra/participant_lookup.go

```diff
@@ -0,0 +1,40 @@
+// Copyleft (ɔ) 2017 The Caliopen contributors.
+// Use of this source code is governed by a GNU AFFERO GENERAL PUBLIC
+// license (AGPL) that can be found in the LICENSE file.
+
+package store
+
+import (
+	"fmt"
+	. "github.com/CaliOpen/Caliopen/src/backend/defs/go-objects"
+	"github.com/gocassa/gocassa"
+	"strings"
+)
+
+func (cb *CassandraBackend) RetrieveParticipantLookup(user_id, identifier, typ string) (lookup *ParticipantLookup, err error) {
+
+	// retrieve participant lookup
+	lookup = new(ParticipantLookup)
+	m := map[string]interface{}{}
+	clean_id := strings.ToLower(identifier)
+	q := cb.SessionQuery(`SELECT * FROM participant_lookup WHERE user_id = ? AND identifier = ? AND type = ?`, user_id, clean_id, typ)
```

Comment:
should we able to query on `user_id` and `identifier` only ? (`typ` optional)

Reply:
We can query this way but it's a collection retrieved and must be a distinct function